packages/react-components/package.json

```diff
@@ -13,15 +13,18 @@
   "license": "MIT",
   "scripts": {
     "build": "just-scripts build",
-    "bundle": "just-scripts bundle",
-    "bundle:storybook": "just-scripts storybook:build",
     "bundle-size": "bundle-size measure",
-    "chromatic": "npx chromatic@5.6.3 --project-token $CHROMATIC_PROJECT_TOKEN --exit-zero-on-changes --build-script-name bundle:storybook",
+    "chromatic": "npx chromatic@5.6.3 --project-token $CHROMATIC_PROJECT_TOKEN --exit-zero-on-changes --build-script-name build-storybook",
```

Comment:
with this we will ship a proper production builds (`build-storybook` sets ENV to `production`